packages/react/src/TooltipV2/Tooltip.examples.stories.tsx

```diff
@@ -161,7 +161,7 @@ export const DialogTrigger = () => {
   return (
     <>
       <Tooltip text="Ready to merge">
-        <IconButton ref={buttonRef} onClick={() => setIsOpen(!isOpen)} icon={CheckIcon} aria-label="Check mark" />
+        <IconButton ref={buttonRef} onClick={() => setIsOpen(!isOpen)} icon={CheckIcon} aria-label="Ready to merge" />
```

Comment:
small nit, but otherwise good to go! 

Could we change the label here to "merge"? This is because the tooltip's text is the same label which causes it to announce twice

Reply:
Yes! thanks for pointing that out 🙏🏼